spryker/setup_suite.sh

```diff
@@ -84,32 +104,7 @@ OLD_APPLICATION_VERSION=$(readlink /data)
 sudo rm -rf /data
 ln -s $APPLICATION_PATH /data
 sudo rm -rf $OLD_APPLICATION_VERSION
-sudo chown jenkins /versions
-
-#Cron jobs generate
-updateCronJobs() {
-   appHost=$1
-   cronJobFile="vendor/spryker/setup/src/Spryker/Zed/Setup/Business/Model/Cronjobs.php"
-   if [ -f ${cronJobFile} ]; then
-      cd $APPLICATION_PATH
-      ## remove it after march release 
-      sed -i 's/\\\$destination_release_dir/$destination_release_dir/g' ${cronJobFile}
-      ##
-      sed -i 's/\$PHP_BIN//g' config/Zed/cronjobs/jobs.php
-      patch -p0 < /etc/spryker/Cronjobs.patch
-      sed -i "s/appHost/${appHost}/g" ${cronJobFile}
-      vendor/bin/console setup:jenkins:generate
-   fi
-}
-
-# if local build
-if [ "${ZED_HOST}" == "os.de.demoshop.local" ]; then
-   updateCronJobs "app"
-# if build run  on an AWS instance 
-elif $(nc -znw 2 169.254.169.254 80); then
-   instance_ip=$(curl http://169.254.169.254/latest/meta-data/local-ipv4)
-   updateCronJobs ${instance_ip}
-fi
+sudo chown www-data /versions
```

Comment:
-R ?

Reply:
No need for -R since 6 lines before we do recursive chown for APPLICATION_PATH.

Moreover, -R on /versions folder can take much longer since contains all previous versions of application.